.github/workflows/build-and-publish.yml

```diff
@@ -61,7 +61,7 @@ jobs:
 
       # Using msbuild to build and package the solution.
       - name: Build
-        run: msbuild -t:build -restore -verbosity:m -property:Configuration=Release
+        run: msbuild ${{ needs.set_variables.outputs.PACKAGE_NAME }}/${{ needs.set_variables.outputs.PACKAGE_NAME }}.csproj -t:build -restore -verbosity:m -property:Configuration=Release
```

Comment:
Was it finding a different csproj before or something?

Reply:
@NeilSchiefer Without specifying the project it build everything, which then causes issues for the snyk monitor step.  Also, it speeds up the pipeline since projects that aren't being released aren't built.  The problem is, when testing the switch to msbuild, I tested with a single project repo, so it didn't come up.  Previously, when using dotnet build, we were specifying the project like this.  When I pasted in the msbuild command and flags I missed adding that back in.